draft-ietf-masque-connect-ip.md

```diff
@@ -839,26 +865,20 @@ packet is transmitted in an HTTP Datagram. This prevents infinite loops in the
 presence of routing loops, and matches the choices in IPsec {{?IPSEC=RFC4301}}.
 This does not apply to IP packets generated by the IP proxying endpoint itself.
 
-Note that the IP forwarding tunnels described in this document are not
-fully featured "interfaces" in the IPv6 addressing architecture sense
-{{?IPv6-ADDR=RFC4291}}. In particular, they do not necessarily have
-IPv6 link-local addresses. Additionally, IPv6 stateless
-autoconfiguration or router advertisement messages are not used in
-such interfaces, and neither is neighbor discovery. Implementers need
-to ensure that they do not forward any link-local traffic beyond the
-IP proxying interface that it was received on. IP proxying endpoints
-also need to properly reply to packets destined to link-local
-multicast addresses.
+Implementers need to ensure that they do not forward any link-local traffic
+beyond the IP proxying interface that it was received on. IP proxying endpoints
+also need to properly reply to packets destined to link-local multicast
+addresses.
 
 IPv6 requires that every link have an MTU of at least 1280 bytes {{IPv6}}.
 Since IP proxying in HTTP conveys IP packets in HTTP Datagrams and those can in
 turn be sent in QUIC DATAGRAM frames which cannot be fragmented
 {{!DGRAM=RFC9221}}, the MTU of an IP tunnel can be limited by the MTU of the
 QUIC connection that IP proxying is operating over. This can lead to situations
-where the IPv6 minimum link MTU is violated. IP proxying endpoints that support
-IPv6 MUST ensure that the IP tunnel link MTU is at least 1280 (i.e., that they
-can send HTTP Datagrams with payloads of at least 1280 bytes). This can be
-accomplished using various techniques:
+where the IPv6 minimum link MTU is violated. IP proxying endpoints that operate
+as routers and support IPv6 MUST ensure that the IP tunnel link MTU is at least
+1280 (i.e., that they can send HTTP Datagrams with payloads of at least 1280
+bytes). This can be accomplished using various techniques:
```

Comment:
This one is a bit complex. In some sense it it the link responsibility to ensure that it supports a MTU of at least 1280. At the same time we have put the verification aspect onto the router to verify that an 1280 bytes MTU encapsulated packet will make it through. So the first bullet below still works. But, the second ones creates this strange interaction where the router needs to ping the other end of the tunnel, and if the min MTU ping fails consistently then withdraw the route until it succeeds.

Reply:
I agree with you. I don't see a better solution though, so I think we'll have to live with this bit not fitting in perfectly